src/cljs/athens/listeners.cljs

```diff
@@ -220,17 +220,19 @@
     EventType.BEFOREUNLOAD
     (fn [e]
       (let [synced? @(subscribe [:db/synced])
+            isEditing? @(subscribe [:editing/uid])
```

Comment:
`isEditing?` is a mix between JS and CLJ naming.
In Clojure we'd just say `editing?`.
Change this and we're merging.

Reply:
Got it, fixed now. Ready for merging i think

Reply:
@sawhney17 🙇‍♂️